mobile/lib/main.dart

```diff
@@ -119,12 +122,16 @@ class _TenTenOneAppState extends State<TenTenOneApp> {
 
   @override
   Widget build(BuildContext context) {
+    MaterialColor swatch = Colors.blue;
+
     return MaterialApp.router(
       title: "10101",
       theme: ThemeData(
-        primarySwatch: Colors.blue,
-        extensions: const <ThemeExtension<dynamic>>[
-          TradeTheme(),
+        primarySwatch: swatch,
+        useMaterial3: true,
```

Comment:
💡 Interesting, I did not know one has to configure that to use the latest version.
❓What's the effect of setting this? Are there any compatibility concerns here, i.e. is material3 only supported by android >x ?

Reply:
This was a quick and dirty solution to get the send/receive buttons working as I copied them from a flutter example which relies on the material3 ColorScheme properties being set up properly. It can definitely be changed but I am not 100% sure how we want to style the buttons. However, apparently Flutter will move to having useMaterial3: true as default soon anyway [(see docs(](https://api.flutter.dev/flutter/material/ThemeData/useMaterial3.html). Apparently it may lead to an inconsistent look and feel for unmigrated widgets.

Reply:
Ok, thanks for the link. 
It seems that the transition from m2 to m3 is still in progress - but mostly for the m3.web components, as far as I could see the mobile components are stable already.

---

I think it's OK to stick to m3 and accept that some widgets might not behave perfect; m2 will not be supported eventually, so better stick to m3 early.